### ISSUE TITLE
Cannot add user to local wandb

### ISSUE THREAD

**Issue by hokmund** [wandb/client#1056]
Weights and Biases version: version 0.8.36
Python version: 3.6.9
Operating System: Ubuntu 16.04

### Description

After setting up local wandb I go to `http://<host>:8080/admin/users` and try to add another user via "Add user" button.
I expect it to generate an invitation link for a new user.

### What I Did

When I press submit button, I get an error message:
```
only admins can use createUser()
```
There are two users in the table. The first one is the default user that has the same name as my ubuntu account. The second one is the user I created after I set up wandb client.

![wandb](https://user-images.githubusercontent.com/32386791/82642600-daa9dc00-9c16-11ea-84d7-2f2f549c1623.png)

**Comment by vanpelt** [comment]
You can open the Chrome developer console and run the following command to get admin access.  We're working on a new release to fix this that should go out in the next couple days:

```
document.cookie = "use_admin_privileges=true;"
```

**Comment by hokmund** [comment]
Thanks!
It worked for me.

I guess, I shouldn't close the issue though, so you will be able to link the fix.

**Comment by cvphelps** [comment]
Fixed in wandb/local:0.9.16
